jquery.xblock.js

```diff
@@ -161,22 +167,23 @@
 
             $('.xblock', root).not('.xblock-initialized').each(function(index, blockDOM) {
                 var initFnName = $(blockDOM).data('init');
-                if (initFnName) {
-                    // Don't fail when the page still contains XModules
-                    if (initFnName === 'XBlockToXModuleShim') {
-                        console.log('Warning: Unsupported XModule JS init', blockDOM);
-                        return;
-                    }
+                if (!initFnName) {
```

Comment:
@e-kolpakov This might be part of why you're having issues with OfficeMix, since this change (and any not directly related to the notifications) came from you-know-where.